src/main/cli/cli.c

```diff
@@ -3593,6 +3593,48 @@ static void cliGpsPassthrough(const char *cmdName, char *cmdline)
 
     gpsEnablePassthrough(cliPort);
 }
+
+#ifdef USE_GPS_DEBUG
+static void cliGpsInfo(const char *cmdName, char *cmdLine) {
+    UNUSED(cmdName);
+    UNUSED(cmdLine);
+
+    cliPrintLinef("# MON-VER acquired: %s", gpsData.unitVersion != UBX_VERSION_UNDEF ? "true" : "false");
+
+    if (gpsData.unitVersion != UBX_VERSION_UNDEF) {
+
+        cliPrintLinef("swVersion: ");
+        cliPrint(" ");
+        cliPrintLinef("FW=%d.%d, PROTO=%d.%d",
+                      gpsData.monVer.swVersion.firmwareVersion.major, gpsData.monVer.swVersion.firmwareVersion.minor,
+                      gpsData.monVer.swVersion.protocolVersion.major, gpsData.monVer.swVersion.protocolVersion.minor
+        );
+
+        cliPrintLinef("hwVersion: ");
+        cliPrint(" ");
+        cliPrint(ubloxVersion_map[gpsData.unitVersion].str);
+        cliPrint(" (");
+        cliPrintf("%08X", gpsData.monVer.hwVersion);
+        cliPrintLine(")");
```

Comment:
Perhaps this is resolved, because this has been simplified to only show the Hardware version.

Reply:
Yes, the cli GPS reporting was limited to the line in the `status` command, which was simplified to just show the platform version (e.g., M8), and the additional `gps_status` command was removed since all it did was duplicate the same info.